terraform/main.tf

```diff
@@ -204,6 +211,24 @@ data "aws_iam_policy_document" "read_datadog_api_key_secret" {
   }
 }
 
+module "grants_prepared_dynamodb_table" {
+  source  = "cloudposse/dynamodb/aws"
+  version = "0.32.0"
+  context = module.this.context
+
+  name             = "prepareddata"
+  hash_key         = "grant_id"
+  table_class      = "STANDARD"
+  enable_streams   = true
+  stream_view_type = "NEW_AND_OLD_IMAGES"
+  # enable_point_in_time_recovery  = true
+  enable_encryption = true
+}
+
+# resource "aws_dynamodb_contributor_insights" "grants_prepared_dynamodb_main" {
+#   table_name = module.grants_prepared_dynamodb_table.table_name
+# }
```

Comment:
Should either be uncommented or removed. Do you think it would be worthwhile for us to enable contributor insights for this table?

Reply:
Oh, this is commented out because Localstack doesn't appear to support this resource yet.  It causes terraform to error out whenever I try to apply it.  Not sure what the best course of action is here since this will likely apply just fine against real AWS.

Reply:
(Replied [here](https://github.com/usdigitalresponse/grants-ingest/pull/46/files#r1176956276))